### ISSUE TITLE
Move issues from  old repository

### ISSUE THREAD

**Issue by nloyola** [emacs-eclim/emacs-eclim#13]
Should we move the open issues from [senny/emacs-eclim](https://github.com/senny/emacs-eclim/issues) to this repository?

It can be done using this [tool](https://github.com/IQAndreas/github-issues-import).

**Comment by gopar** [comment]
Yeah, I don't mind moving issues. We just need to check if they can be reproduced.

**Comment by vspinu** [comment]
It would be already good enough if people could fix issues and merge pulls in current project. There was quite some enthusiasm about this new fork last year, but now the history seems to be repeating itself. Last commit is from 15th of November and pulls seem to be piling again.
